### DISCUSSION TITLE
[auth] Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by JantHsueh** [community/community#118368] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I subscribed copilot for one year, and in the bill, I can see a record of successful payment. But when I downloaded the plug-in and logged in to github in android studio, it prompted "You don't have access to GitHub Copilot. Sign up by visiting"
![image](https://github.com/community/community/assets/13702466/eb022cbb-7e38-4e6b-8432-60966f357ba1)

I tried using an old version of the plugin and deleting the ~/.config/github-copilot folder, but it still didn't work.

I checked the log of android studio, idea.log, and there is this prompt

2024-04-08 09:55:42,946 [54141713] INFO - #copilot - [auth] Invalid copilot token: missing token: 403
2024-04-08 09:56:19,606 [54178373] INFO - #copilot - [auth] Invalid copilot token: missing token: 403

I submitted a ticket, but have not received a reply or resolution.  https://support.github.com/ticket/personal/0/2717668 . My payment has been successful for 3 days, but it still cannot be used normally.

Please help me solve the problem as soon as possible and compensate for the losses in the past few days

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
